modules/experimental/pages/destiny-not-fate.adoc

```diff
@@ -0,0 +1,16 @@
+= Destiny Not Fate
+
+`Unit ability`
+
+{blank}[TODO: add introductory text.]
+
+---
+
+Where your unit has the {destiny-not-fate} ability:
+
+* You can use Psychic Blast Fate cards as if the unit were a {psyker}, but instead of inflicting a hit, remove Blast markers from the unit's detachment. When the card is used, remove as many Blast markers as the Psychic Blast's value.
```

Comment:
To neaten and disambiguate, and also to mesh with the new Fate card texts I'm working on:

```suggestion
* This unit allows you to use 'Psychic Attack (X) — Psychic Blast' Fate cards in a special way.
Instead of the usual effect, remove (X) number of Blast markers from the unit's detachment.
```

Reply:
I haven't seen the new text, but from what I can infer from the structure this looks better.

Just bear in mind that the ability text has to work with the Epic 40k Fate cards until the new ones have been edited, laid out, and published. But players should be able to work out how this text should apply to existing cards!

Reply:
A valid concern, but yes, I thought about that. You probably get this, but to make sure we're on the same page:

As you know, the later Fate cards added more types of psychic attack — and some card effects on these cards mean to refer to psychic attacks and their power levels in general, while others refer to a specific type of psychic attack. I found that they don't always disambiguate well, so references were harder to formulate clearly than they should be.

The 'Psychic Attack (X) — {name of effect}' format makes it easy to disambiguate. At the same time, it's so obvious how the old cards with the '{name of effect (X)}' format should interoperate with this new format that I'd be very surprised if it caused problems.

Reply:
Like I said, 'But players should be able to work out how this text should apply to existing cards!'.

Reply:
As I said:

> it's so obvious how the old cards with the '{name of effect (X)}' format should interoperate with this new format that I'd be very surprised if it caused problems

So, what's the problem you see that I don't? If a person can't 'work out' that Remastered 'Psychic Attack (3) — Psychic Blast' and original 'Psychic Blast (3)' are interoperable, then they probably aren't equipped for tabletop wargames in the first place, are they...?

Reply:
Lol, a classic example where a statement can be read in two very different ways! 😂 Cool, I'll apply the suggestion.